apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -40,6 +41,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// Container image fall-back defaults
+
+	// RabbitMqContainerImage is the fall-back container image for RabbitMQ
+	RabbitMqContainerImage = "registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0"
```

Comment:
I think we should change this to `quay.io/podified-antelope-centos9/openstack-rabbitmq:current-podified`

Reply:
Ack, will do